mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
basically it's an interaction between the figure size and the scaling. If the figure size is huge, scale < 1 won't matter